### PR TITLE
Fix wrong filename

### PR DESCRIPTION
**Pull request by migo315**
Change doctrine.yaml to framework.yaml.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@migo315 thanks for this ... but before merging I'm going to ask to @xabbuh, one of our doc maintainers, to see if he agree with this. It's tricky because you are configuring the session with Doctrine ... so, should it go to `doctrine.yaml` or `framework.yaml`?

**Comment by xabbuh** [comment]
As this part is configuring the FrameworkBundle (where this option is already present in the current version of the recipe: https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/config/packages/framework.yaml#L9-L10) I agree with @migo315.

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks @migo315.

**Pull request merged**
